cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -382,7 +387,11 @@ data ScriptExecutionError =
 
        -- | An attempt was made to spend a key witnessed tx input
        -- with a script witness.
-     | ScriptErrorNotPlutusWitnessedTxIn ScriptWitnessIndex
+     | ScriptErrorNotPlutusWitnessedTxIn ScriptWitnessIndex ScriptHash
+
+       -- | A redeemer was supplied which points to a script hash
```

Comment:
What does it mean "cannot resolve"? This is vague. The script does not exist in the transaction?

Reply:
The helper function attempts to glue a bunch of stuff together. In particular, it tries to take the redeemer pointers, connect them to a script hash, and then figure out what script corresponds to that script hash based on all the information it has in the tx (reference scripts and witnesses outside the body). This error happens if the pointer points to a script hash that we cannot identify.

Reply:
works for me!